### ISSUE TITLE
Old models never die

### ISSUE THREAD

**Issue by darkpixel** [mmcardle/django_builder#91]
I was signed in as an anonymous account.  When I removed all my projects and created a new one with a new name, the old models populated the top of the screen.

**Comment by mmcardle** [comment]
Hi @darkpixel 
Are these models you have imported from a models.py file? 
The imported models are not attached to particular project so they persist, they can be deleted from the import screen.

**Comment by darkpixel** [comment]
Yeah, I imported them from a models.py file.  There were literally somewhere around 300 models in this project.  We basically introspected a 3rd-party database and we use Django to access the data.  I was hoping to auto-generate a bunch of the boilerplate using django_builder, but it's forcing me to add every model by hand into an application.  Maybe I'm missing something...

**Comment by mmcardle** [comment]
Ah yes I see how that could be frustrating.
To be honest I have only imported around 5 models max to test.

The flow is meant to be ->
* Upload a models.py file, any models found appear in a bar along the top.
* View the models in a dialog, choose which models to import and add the to an app.

What do you think would help in your situation (more than 1 might apply)
* Add all models found automatically to an app?
* Add  "Add All" and "Delete All" buttons to the import dialog?
* Remove all imported models from the dialog when you create a new project.

Thanks for your feedback

**Comment by darkpixel** [comment]
Sorry--I'm not ignoring you.  It's just been a busy couple of weeks.

I think those would all work, but I think it would be more efficient to upload directly to an app.

Basically models aren't a "project level" item.  They are application level.  When importing, they should probably be imported against an app directly without having to manually associate them.

I suppose someone might want to break things up into multiple apps, but I think that could be handled if necessary by being able to right-click a model and have an option to "copy/move to app" or something like that.

**Comment by mmcardle** [comment]
That makes total sense, and would tidy up the UI quite a bit.

**Comment by mmcardle** [comment]
@darkpixel this should now be fixed, there is an `Upload` button when adding a model to an app.

**Comment by darkpixel** [comment]
Nice!  I just checked out the live site.
Would it be too much to ask to have an "Add All" button on the screen that comes up after uploading models?  I want to load ~150 models, and don't want to keep clicking the "Add" button and waiting ~1 second for the page to refresh. :)

**Comment by mmcardle** [comment]
@darkpixel 
Agreed and done.

**Comment by darkpixel** [comment]
It spent several minutes trying to import a bunch of models, and tossed this into dev tools:

![Screenshot from 2020-07-23 14-23-33](https://user-images.githubusercontent.com/158380/88340204-44666500-ccf0-11ea-8b95-3b51b9f19ce2.png)

**Comment by mmcardle** [comment]
@darkpixel Will see if i can fix that, might have to do them in batches.
How many models and fields in total are you importing?

**Comment by darkpixel** [comment]
196

**Comment by mmcardle** [comment]
The issues with importing so many models is hitting the quota limits of the backend system (firebase).
Currently the free tier has hard limits when you get to the quota https://firebase.google.com/pricing .
